### PR TITLE
Add brettmc to approvers

### PR DESCRIPTION
**Pull request by jack-berg**
Thanks for all your contributions so far @brettmc!

Upon merging this PR, I'll create the `configuration-approvers` team.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,3 +69,22 @@ When a property requires pattern matching, use wildcard `*` (match any number of
...
 
 In instances where there is a type mismatch between the JSON schema and equivalent standard env var, an alternative version of the property may be provided to resolve the mismatch. For example, resource attributes are configured at `.resource.attributes`, but `.resource.attributes_list` is available with a format matching that of `OTEL_RESOURCE_ATTRIBUTES`. Alternative properties are reserved for cases where there is a demonstrated need for platforms to be able to participate in configuration and there is no reasonable alternative.
+
+## Contributing
+
+See [CONTRIBUTING.md](CONTRIBUTING.md)
+
+Approvers ([@open-telemetry/configuration-approvers](https://github.com/orgs/open-telemetry/teams/configuration-approvers)):
+
+- [Brett McBride](https://github.com/brettmc), TODO
```
Comment (jack-berg) [inline README.md:79]: @brettmc do you have a company affiliation you want to add?

**Comment by brettmc** [comment]
> do you have a company affiliation you want to add?

`Deakin University`

**Comment by codeboten** [comment]
Thanks @brettmc!

File: README.md (hunk 1 above)
Comment (codeboten) [inline README.md:79]:
```suggestion
- [Brett McBride](https://github.com/brettmc), Deakin University
```

**Review by codeboten — APPROVED** [review]

**Comment by jack-berg** [comment]
`configuration-approvers` created and granted write permissions to this repository.

**Pull request merged**